app/views/layouts/_user_config.html.erb

```diff
@@ -1,9 +1,13 @@
 <script>
   try {
-    const bodyClass = localStorage.getItem('config_body_class');
+    var bodyClass = localStorage.getItem('config_body_class');
     const userString = localStorage.getItem('current_user');
 
     if (bodyClass) {
+      if (document.body.classList.contains('hamburger-open')) {
+        bodyClass += ' hamburger-open';
+      }
+
```

Comment:
_Please read the PR description if you haven't already as it explains why this is here._

Part of the challenge of the current setup is the entire "hamburger-open" logic lives on the `body`. Admittedly, this proposed fixed here does feel a little...hacky?

Another avenue is to move the menu logic off of the `body` element to avoid having to account for this here. I'm worried we'll run into this again though where we have classes on `body` and forget about this setup 🤔 . Maybe there's a better solution?

Reply:
Refactor (suggestion): Instead of a CSS class, we can use a `data` attribute, e.g. `data-sidebar-menu-state="open"` or something along those lines. And then it will remove the need for the hack. Gonna pull this locally and take it for a test drive.